website/docs/opengl/c1/3-hello-texture.md

```diff
@@ -0,0 +1,464 @@
+---
+{
+    "TableOfContents": {
+        "Name": "1.3 - Hello Texture",
+        "Url": "3-hello-Texture.html",
+        "Metadata": {
+            "AuthorGitHub": "lumi2021",
+            "DateTimeWritten": "11/01/2024 16:00",
+            "PreviewImage": ""
+        }
+    }
+}
+---
+
+# 1.3 - Hello Texture
+<?# Info "You can view the source code for this tutorial [here](../sources/1.3-final-result.html). This tutorial builds on the previous tutorial. If you haven't read it, you can do so [here](1-hello-window.html)." /?>
+
+In the previous tutorial, we've shown you how to open an OpenGL window and draw a colored quad.
+                                                    
+Another very important thing in graphics programming is the ability to draw images. In this tutorial, you'll learn:
+
+* What a texture is.
+* How to send more data to shaders.
+* How to use texture coordinates.
+* How to import images as textures.
+* How shaders handle images.
+* How OpenGL handle transparency.
+* How to use texture parameters.
+* What are mipmaps and how to use them.
+
+## What a texture is
+Technically, textures are multidimensional objects that store color data.
+
+The most common type of texture is the 2D texture, which stores a 2D grid of pixels of an image (this is just for comparison.
+Texture objects are a lot more complex than just 2D arrays!).
```

Comment:
```suggestion
The most common type of texture is the 2D texture, which stores a 2D grid of pixels of an image (This is just for comparison,
texture objects are a lot more complex than just 2D arrays!).
```

Reply:
I disagree with this comment, the parenthesised expression should not be capitalised as it is still part of the sentence.

Reply:
Is that how that works? I thought if the text within the parentheses was a full sentence on its own you capitalized the text within. 
How i read this is that it's two separate sentences "The most common type of texture is the 2D texture, which stores a 2D grid of pixels of an image" and "This is just for comparison,
texture objects are a lot more complex than just 2D arrays", with just parentheses making the intended relationship between them clearer.

Reply:
A parenthesised expression should not be its own sentence in most cases. Parentheses are used to enclose additional, non-essential information to clarify, explain, or add a side note in a sentence. Use parentheses to prevent disrupting the flow of a sentence.

Reply:
So it sounds like a standalone sentence may be appropriate here.

Reply:
I see the point for this and agree with the change, but I don't think the first word inside the parenthesis should be capitalized

Reply:
Updated the suggestion to reword it